GOVERNANCE.md

```diff
@@ -0,0 +1,219 @@
+
+## Committer Nominations
+
+Any existing Committer can nominate an individual making significant and
+valuable contributions to the OpenColorIO project to become a new Committer.
+
+To nominate a new Committer, open an issue in the OCIO repository, with a
+summary of the nominee's contributions.
+
+If there are no objections raised by any Committers one week after the issue is
+opened, the nomination will be considered as accepted. Should there be any
+objections against the nomination, the TSC is responsible for working with the
+individuals involved and finding a resolution. The nomination must be approved
+by the TSC, which is assumed when there are no objections from any TSC members.
+
+Prior to the public nomination, the Committer initiating it may seek feedback
+from other Committers in private channels and work with the nominee to improve
+the nominee's contribution profile, in order to make the nomination as
+frictionless as possible.
+
+If individuals making valuable contributions do not believe they have been
+considered for a nomination, they may log an issue or contact a Committer
+directly.
+
+## Technical Steering Committee
+
+A subset of the Committers forms the Technical Steering Committee (TSC), which
+has final authority over this project. As defined in the project charter, TSC
+responsibilities include, but not limited to:
+
+* Coordinating technical direction of the Project
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Committers
+* Appointing representatives to work with other open source or open standards
+communities
+* Discussions, seeking consensus, and where necessary, voting on technical
+matters relating to the code base that affect multiple projects
+* Coordinating any marketing, events, or communications regarding the project
+
+Within the TSC are three elected leadership roles to be held by its members and
+voted on annually. Any TSC member can express interest in serving in a role, or
+nominate another member to serve. There are no term limits, and one person may
+hold multiple roles simultaneously. Should a TSC member resign from a leadership
+role before their term is complete, a successor shall be elected through the
+standard nomination and voting process to complete the remainder of the term.
+The leadership roles are:
+
+* Chair: This position acts as the project manager, organizing meetings and
+providing oversight to project administration.
+
+* Chief Architect: This position makes all the final calls on design and
+technical decisions, and is responsible for avoiding "design by committee"
+pitfalls.
+
+* ASWF TAC Representative: This position represents the project at all ASWF
+(Academy Software Foundation) TAC (Technical Advisory Council) meetings.
+
+### TSC Leaders
+
+* Chair: TBD
+* Chief Architect: TBD
+* ASWF TAC Representative: TBD
```

Comment:
As I think about it, maybe we just want have the TSC chair and the chief architect, and say that the TSC can choose which of them should be the TAC representative. Especially for a development community so small, it's not clear to me why we would ever want the TAC representative to be yet a third person who is neither the architect nor the chair. Thoughts?

Reply:
That seems logical. Is there a requirement for the TAC rep to be one static individual, or can it be *a* TSC member, depending on the current availability or needs of the project?